### ISSUE TITLE
Calling package_coverage() results in Error

### ISSUE THREAD

**Issue by thatsIch** [jimhester/covr#246]
I am trying to get `covr` running on:

* Windows 10 x64
* R 3.3.2
* RStudio 1.0.44

After calling:

    > library(covr)
    > package_coverage()
    Error in objs[[1]] : subscript out of bounds

Tests are running through on Travis and locally, but neither code coverage.

Cheers
   Ich

**Comment by jimhester** [comment]
For what package?

**Comment by thatsIch** [comment]
We spun up a project just recently

https://github.com/snaketron/BcellNet

so if there is something fundamentally wrong, please state it. I tried to follow the instructions from  https://www.rstudio.com/resources/webinars/covr-bringing-test-coverage-to-r/

**Comment by thatsIch** [comment]
I just created a new project and I can't reproduce this there. Is it possible that the error is sucked up somehow?

https://github.com/thatsIch/r-test-environment

I noticed that after a cold restart of RStudio I can run `package_coverage()` but running `covr::coveralls()` or `covr::codecov()` locally (without a valid key) you get the message:

    Error in objs[[1]] : subscript out of bounds

If you call `package_coverage()` again in the same session, you still get that message.
